src/test/functional/sproc.spec.ts

```diff
@@ -193,208 +184,182 @@ describe("NodeJS CRUD Tests", function () {
+            const { result: retrievedSproc3 } = await container.storedProcedures.upsert(sproc3);
+            const { result: result3 } = await container.storedProcedures.getStoredProcedure(retrievedSproc3.id).execute([{ temp: "so" }]);
+            assert.equal(result3, "aso");
+        });
+    });
 
-                const { result: retrievedSproc } = await client.createStoredProcedure(
-                    TestHelpers.getCollectionLink(true, db, collection), sproc1);
-                const { result: result1, headers: headers1 } = await client.executeStoredProcedure(
-                    TestHelpers.getStoredProcedureLink(true, db, collection, retrievedSproc));
-                assert.equal(result1, "Success!");
-                assert.equal(headers1[Constants.HttpHeaders.ScriptLogResults], undefined);
-
-                let requestOptions = { enableScriptLogging: true };
-                const { result: result2, headers: headers2 } = await client.executeStoredProcedure(
-                    TestHelpers.getStoredProcedureLink(true, db, collection, retrievedSproc), undefined, requestOptions);
-                assert.equal(result2, "Success!");
-                assert.equal(headers2[Constants.HttpHeaders.ScriptLogResults], "The value of x is 1.");
-
-                requestOptions = { enableScriptLogging: false };
-                const { result: result3, headers: headers3 } = await client.executeStoredProcedure(
-                    TestHelpers.getStoredProcedureLink(true, db, collection, retrievedSproc), undefined, requestOptions);
-                assert.equal(result3, "Success!");
-                assert.equal(headers3[Constants.HttpHeaders.ScriptLogResults], undefined);
+    it("nativeApi Should execute stored procedure with partition key successfully name based", async function () {
+        const { result: db } = await client.databases.create({ id: "sample database" });
+        // create container
+        const partitionKey = "key";
 
-            } catch (err) {
-                throw err;
-            }
-        });
+        const containerDefinition = {
+            id: "coll1",
+            partitionKey: { paths: ["/" + partitionKey], kind: DocumentBase.PartitionKind.Hash },
+        };
+
+        const { result: containerResult } = await client.databases.getDatabase(db.id).containers.create(containerDefinition, { offerThroughput: 12000 });
+        const container = await client.databases.getDatabase(db.id).containers.getContainer(containerResult.id);
+
+        // tslint:disable:no-var-keyword
+        // tslint:disable:prefer-const
+        // tslint:disable:curly
+        // tslint:disable:no-string-throw
+        // tslint:disable:no-shadowed-variable
+        // tslint:disable:object-literal-shorthand
+        const querySproc = {
+            id: "querySproc",
+            body: function () {
+                var context = getContext();
+                var container = context.getcontainer();
```

Comment:
This is actually the body for the sproc, which still uses collection/etc. So this needs to be getCollection still.

@kirillg - this is another friction point for collection -> container rename